eng/scripts/Language-Settings.ps1

```diff
@@ -174,3 +180,18 @@ function Update-dotnet-CIConfig($pkgs, $ciRepo, $locationInDocRepo, $monikerId=$
 
   Set-Content -Path $csvLoc -Value $allCSVRows
 }
+
+
+# function is used to auto generate API View
+function Find-dotnet-Artifacts-For-Apireview($artifactDir, $pkgName)
+{
+  $packages = @{}
+
+  # Find all nupkg files in given artifact directory
+  $files = Get-dotnet-Package-Artifacts $artifactDir
+  foreach($f in $files.Where({ $_.Name.StartsWith($pkgName) })) {
```

Comment:
Is StartsWith correct here? What if we have "Azure.Storage" and "Azure.Storage.Blobs" for exmple.

Reply:
My other comment about using the isolated package paths might make this unnecessary as you should probably only have on nupkg file.